atom/renderer/renderer_client_base.cc

```diff
@@ -180,22 +180,26 @@ void RendererClientBase::RenderFrameCreated(
   blink::WebSecurityPolicy::AddOriginAccessWhitelistEntry(
       GURL(kPdfViewerUIOrigin), "file", "", true);
 #endif  // BUILDFLAG(ENABLE_PDF_VIEWER)
+
+  content::RenderView* render_view = render_frame->GetRenderView();
```

Comment:
This method will be called for all frames created in this process, we should make sure that bg color applies only when the top level frame is created, can you add a condition to ensure that. With `RenderViewCreated` it was implicit.

Reply:
👍 I've guarded it to only apply when `render_frame->IsMainFrame()`